core/domain/user_domain.py

```diff
@@ -67,6 +68,8 @@ class UserSettings:
         user_id: str. The unique ID of the user.
         email: str. The user email.
         roles: list(str). Roles of the user.
+        user_has_viewed_lesson_info_modal_once. Flag to check whether the user
```

Comment:
Change it to "Flag to check whether the user has viewed lesson info modal once which shows the progress of the user through exploration checkpoints."

Reply:
Done!